### ISSUE TITLE
User menu position bug

### ISSUE THREAD

**Issue by 4n70w4** [laravel/nova-issues#2048]
- Nova Version: 2.5.0
- Operating System and Version: Windows 10 Enterprise 10.0.18362 build 18362
- Browser type and version: Chrome 77.0.3865.120
### Description

![image](https://user-images.githubusercontent.com/38257723/66915822-9d4af380-f022-11e9-9db4-9607c6f2a9ac.png)

There was no such problem in previous versions.

### Steps To Reproduce

Add many elements to `<ul class="list-reset">` in `resources/views/vendor/nova/partials/user.blade.php`

```
@foreach(range(0, 100) as $step)
<li><a href="" class="block no-underline text-90 hover:bg-30 p-3">{{$step}}</a></li>
@endforeach
```

**Comment by jbrooksuk** [comment]
Confirmed.

**Comment by davidhemphill** [comment]
Fixed in laravel/nova@c4d8a0d01386516b3509ea1f57190be88ca4b0d0
